rollup.config.mjs

```diff
@@ -10,7 +10,9 @@ export default {
 		dir: `dist/${format==='es' ? 'es' : ''}`,
 		exports: 'auto',
 		format,
-		preferConst: true,
+		generatedCode: {
```

Comment:
preferConst is deprecated in rollup3. generatedCode.constBindings is the new way of doing it